examples/advanced/llm_hf/sft_job.py

```diff
@@ -139,11 +151,23 @@ def define_parser():
         default="SFT",
         help="training mode, SFT or PEFT, default to SFT",
     )
+    parser.add_argument(
+        "--quantize_mode",
+        type=str,
+        default=None,
+        help="quantization mode, float16 or blockwise8, default to None (no quantization)",
```

Comment:
if only two or 3 options, you can do something like:
```
parser.add_argument('--quantize_mode', choices=['train', 'test', 'evaluate'], help='Mode of quantization')
```

Reply:
there can be multiple potentially